src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -10,8 +10,8 @@ The resulting ZIP archive is composed of:
 - a `arrets.xml` file containing the description of all stops (Quays and StopPlaces)
 - a `correspondances.xml` file containing all transfers between stops
 - a `calendriers.xml` containing the services
-- a folder for each network containing
-  + TBD
+- a folder `reseau_<network_id_md5>` for each network containing
+  + a `line_<line_id_md5>.xml` containing the description of routes, and service journeys 
```

Comment:
This is a case (one more?) of trying to make data intended to be read by computer, easy to read by a human at the price of technical complexity... I am not really fond of that.
We could do as you propose with something like
`ligne_<clean_line_id>_<line_id_md5>` with:
- `clean_line_id` being only the alphanumerical characters of the `line_id`
- `line_id_md5` the MD5 checksum of the `line_id` to avoid collisions of the previous rule

@patochectp @ArnaudOggy @datanel What do you think?

Reply:
I don't understand the md5. Which "previous rule" are you talking about?

Reply:
The "previous rule" is

> `clean_line_id` being only the alphanumerical characters of the `line_id`

The MD5 ensure you don't have any collision. Imagine these 3 `line_id`:

1. `AB`
2. `A:B`
3. `A/B`

If we keep these identifiers to create the name of the file `ligne_<line_id>`, we will have a problem on Linux with the third one (`/` is not allowed in filename) and on Mac with the second one (`:` is causing problems).

If we replace all problematic characters with `_`, we have to consider all OS and all filesystems (some allows `/`, some allows Unicode, some allows `:`, etc.). But if we do that, the second and third will become `A_B` which creates a collision of filename.

If we only keep the alphanumerical characters, then the 3 id will collide. A MD5 checksum would solve the problem.

Reply:
technically `ligne_<clean_line_id>_<line_id_md5>` is fine for me.

but are you sure you will have lines with similar ids? (AB, A:B..)?
if you have `A_B_md5_1.xml` and `A_B_md5_1.xml`, you don't really know what line it's about. It is a problem?

Would it be better with `ligne_<clean_line_name>_<line_id_md5>`?

Reply:
That is what @prhod proposed. My opinion on that is: adding the `line_name` in the filename is only here to help humans. But we're producing files intended for computers, not for humans. So I'm not very fond to add the complexity of cleaning a name/id for this reason.

That said, if I'm the only one thinking like this, I can totally go with a solution like you and @prhod propose. I'm not sure if `line_id`, `line_name`, or `line_code` should be used. The last one is optional so I wouldn't use it. For humans, maybe `line_name` is more useful?

Reply:
Fixed in 9ecccb0f3807487d2c2fa44b9004fa17769fac96.